### PR TITLE
Create triangle mixin and helper

### PR DESCRIPTION
**Pull request by chrisopedia**
Why is this pull request necessary:

1. we use triangles for everything
2. can be used for dropdown toggles

Changes proposed in this pull request:

- create mixin as js for directional logic
- create helper to showcase
- create mdcss

Screenshot of docs:

image

### CODE REVIEW

File: styles/helpers/triangle.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+```example:html
+<b class="triangle-up"></b>
+<b class="triangle-right"></b>
+<b class="triangle-down"></b>
+<b class="triangle-left"></b>
+```
+*/
+
+.triangle-up {
+@mixin triangle 10px, up;
```
Comment (jgallen23) [inline styles/helpers/triangle.css]: minor, but should these be tabbed in?

File: styles/mixins/triangle.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function( config ) {
```
Comment (jgallen23) [inline styles/mixins/triangle.js]: are we able to make a triangle like this with this mixin?

screenshot 2016-09-05 at 8 44 49 pm

Comment (chrisopedia) [inline styles/mixins/triangle.js]: No, but something like that, I'd rather use an SVG so you can scale it and keep it clean.  But we could add support for it I guess if we need to.

Comment (jgallen23) [inline styles/mixins/triangle.js]: just curious, but why svg?

Comment (chrisopedia) [inline styles/mixins/triangle.js]: Its an icon, so it seems like a perfect use case.  Requires 1 element, scales with the font, changes color with the parent, easier to style (doesn't require 2 selectors).

File: styles/helpers/triangle.css (hunk 1 above)
Comment (chrisopedia) [inline styles/helpers/triangle.css]: probably.

File: styles/mixins/triangle.js (hunk 2 above)
Comment (jgallen23) [inline styles/mixins/triangle.js]: I don't follow.  What do you mean its an icon?  Should just be an `:after` on the selected tab state

Comment (chrisopedia) [inline styles/mixins/triangle.js]: Oh.  You cut off the image then.  It looks like a chat bubble icon.

Comment (chrisopedia) [inline styles/mixins/triangle.js]: We could still probably use CSS transforms to make this possible with the current mixin if need be.

Comment (chrisopedia) [inline styles/mixins/triangle.js]: Or just use a right facing triangle and hide it behind the tab.  That would work.

**Pull request merged**